### PR TITLE
Readme redistribution and reorganisation

### PR DESCRIPTION
**Pull request by r3nor**
I feel like the current Readme is a bit chaotic in some aspects. I tried to redistribute and make it feel more natural visually and easier to read. I hope you like the changes :)

The information in the Readme is exactly the same!

### CODE REVIEW

**Comment by bo0tzz** [comment]
While I'll leave it up to Alex to merge this, I think this is a very welcome bit of tidying up!

File: README.md [hunk 1]
```diff
@@ -173,57 +175,31 @@ Run `docker-compose up` or `docker compose up` (based on your docker's version)
...
-<p align="left">
-  <img src="design/admin-registration-form.png" width="300" title="Admin Registration">
-<p/>
 
-Additional accounts on the server can be created by the admin account.
-
-<p align="left">
-  <img src="design/admin-interface.png" width="500" title="Admin User Management">
-<p/>
+| You can add and manage users from de administration page. | <img src="design/admin-interface.png" width="500" title="Admin User Management"> |
```
Comment (zackpollard) [inline README.md:182]: Typo: "de" should be "the"

File: README.md [hunk 2]
```diff
@@ -14,75 +20,66 @@
...
+  <br/>  
 
-**!! NOT READY FOR PRODUCTION! DO NOT USE TO STORE YOUR ASSETS !!**
+# Screenshots
 
-This project is under heavy development, there will be continuous functions, features and api changes.
+### Mobile
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup tunning </p> | <img src="design/home-screen.jpeg" width="150" title="Home Screen"> <p align="center"> Home Screen </p> | <img src="design/search-screen.jpeg" width="150" title="Curated Search Info"> <p align="center"> Curated search </p> |
```
Comment (zackpollard) [inline README.md:62]: Typo: "Backup tunning" should be "Backup tuning" however I think maybe "Backup Selection" is better?

**Review by zackpollard — CHANGES_REQUESTED** [review]
Looks much cleaner like this, the contents is also a great addition, thanks for the PR. Will also leave this down to Alex to merge in. A couple of typos in there that i've commented on 😄

**Comment by r3nor** [comment]
@zackpollard thank you for the corrections, just fixed them!

File: README.md [hunk 3]
```diff
@@ -14,75 +20,66 @@
...
+| ☑️ Selective album(s) for backup | Yes | N/A
+| ⬇️ Download photos and videos to local device | Yes | Yes
+| 👪 Multi-user support | Yes | Yes
+| 🖼️ Album | Yes | Yes
+| 🤝 Shared Albums | Yes | Yes
+| 🚀 Quick navigation with draggable scrollbar | Yes | Yes
+| 🗃️ Support RAW (HEIC, HEIF, DNG, Apple ProRaw) | Yes | Yes
+| 🧭 Metadata view (EXIF, map) | Yes | Yes
+| 🔎 Search by metadata, objects and image tags | Yes | No
+| ⚙️ Administrative functions (user management) | N/A | Yes
```
Comment (alextran1502) [inline README.md:52]: I love the emoji here, but it will be hard to find the applicable one for new features

File: README.md [hunk 4]
```diff
@@ -14,75 +20,66 @@
...
+  <br/>   
 </p>
 
-# Immich
+## Content
+- [Features](#features)
+- [Screenshots](#screenshots)
+- [Installation](#installation)
+- [Mobile App](#-mobile-app)
+- [Developers](#development)
```
Comment (alextran1502) [inline README.md:31]: This would be `Development`?

File: README.md [hunk 5]
```diff
@@ -134,10 +136,10 @@ wget -O .env https://raw.githubusercontent.com/immich-app/immich/main/docker/.en
 
 ### Step 2 - Populate .env file with customed information
 
-* Populate customised database information if necessary.
+* Populate custom database information if necessary.
 * Populate `UPLOAD_LOCATION` as prefered location for storing backup assets.
-* Populate a secret value for `JWT_SECRET`
-* [Optional] Populate Mapbox value.
+* Populate a secret value for `JWT_SECRET`, you can use this command: `openssl rand -base64 128`
```
Comment (alextran1502) [inline README.md:141]: Great addition, thank you

File: README.md [hunk 6]
```diff
@@ -173,57 +175,31 @@ Run `docker-compose up` or `docker compose up` (based on your docker's version)
...
 
-<p align="left">
-  <img src="design/google-play-qr-code.png" width="200" title="Google Play Store">
-<p/>
 
-## iOS
+  <br/>  
 
-#### Get the app on Apple AppStore [here](https://apps.apple.com/us/app/immich/id1613945652):
+# 🤳 Mobile app
```
Comment (alextran1502) [inline README.md:193]: Let's keep the header consistent with other headers that don't have the emoji

File: README.md [hunk 7]
```diff
@@ -14,75 +20,66 @@
...
+# Screenshots
 
-This project is under heavy development, there will be continuous functions, features and api changes.
+### Mobile
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup selection </p> | <img src="design/home-screen.jpeg" width="150" title="Home Screen"> <p align="center"> Home Screen </p> | <img src="design/search-screen.jpeg" width="150" title="Curated Search Info"> <p align="center"> Curated search </p> |
+| <img src="design/shared-albums.png" width="150" title="Shared Albums"> <p align="center"> Shared albums </p>  |  <img src="design/nsc6.png" width="150" title="EXIF Info"> <p align="center"> EXIF info </p>  | <img src="https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif" width="150" title="Loading ~4000 images/videos"> <p align="center"> Loading ~4000 images/videos </p>  |
```
Comment (alextran1502) [inline README.md:63]: image
I think the table format here makes the layout kinda hard to follow as the indentation offset for each cell

Comment (alextran1502) [reply README.md:63]: let's try a different ideas of display these messages or we can keep it linearly

**Comment by alextran1502** [comment]
Thank you for a great PR, I added some suggestion, let me know if you have any question

File: README.md (hunk 3 above)
Comment (r3nor) [reply README.md:52]: So you think it is better to remove them?

File: README.md (hunk 4 above)
Comment (r3nor) [reply README.md]: Yes, just fixed it!

File: README.md (hunk 7 above)
Comment (r3nor) [reply README.md:63]: I just changed it to a linear display with centered images. Maybe you think this is better.

Also removed the `.env` example text and replaced it with a link (that opens in a new tab) to the `.env.example` file that is present in the repo. This way you don't need to change the `.env` in two places and users are linked directly to the file itself. Also this optimizes the space on the readme so steps look more simple at a glance.

**Comment by alextran1502** [comment]
Thanks, it looks good now
